From 2bdc384b77d0a06227e8a893fbbd4f47d413d421 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 May 2014 14:36:08 +0200
Subject: [PATCH] cms/cmssetup.sh: convert SUBCHANNELS to lowercase

In udev rules, the channels need to be expressed in lower case.
---
 modules.d/80cms/cmssetup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/80cms/cmssetup.sh b/modules.d/80cms/cmssetup.sh
index 36955bc3b4..c9e1331466 100755
--- a/modules.d/80cms/cmssetup.sh
+++ b/modules.d/80cms/cmssetup.sh
@@ -144,6 +144,7 @@ function dasd_settle_all() {
     processcmsfile()
     {
         source /tmp/cms.conf
+        SUBCHANNELS="$(echo $SUBCHANNELS | sed 'y/ABCDEF/abcdef/')"
 
         if [[ $NETTYPE ]]; then
 	    (
-- 
1.9.3

